modules/opa/download_policy.go

```diff
@@ -66,9 +62,32 @@ func DownloadPolicyE(t testing.TestingT, rulePath string) (string, error) {
 	tempDir = filepath.Join(tempDir, "getter")
 
 	logger.Default.Logf(t, "Downloading %s to temp dir %s", rulePath, tempDir)
-	if err := getter.GetAny(tempDir, baseDir); err != nil {
+	if _, err := getter.GetAny(context.Background(), tempDir, baseDir); err != nil {
 		return "", err
 	}
 	policyDirCache.Store(baseDir, tempDir)
 	return filepath.Join(tempDir, subDir), nil
 }
+
+// detectGetter find suitable getter.Getter by rulePath.
+func detectGetter(rulePath string) (getter.Getter, error) {
```

Comment:
Would you please create unit test for this change to make sure we can continue to validate whether this code works going forward and now?

Reply:
I thought about it a bit and rewrote the patch. This variant should be more straightforward.